README.md

```diff
@@ -1,2 +1,137 @@
 # Starcounter.Uniform
-Helping library for server-side part of uniform components.
+Starcounter.Uniform is a helping library that provides server-side helpers for using and managing uniform components. 
+
+It is available for downloading as [Starcounter.Uniform](https://www.nuget.org/packages/Starcounter.Uniform/) NuGet package.
+
+## Requirements
+Requires Starcounter 2.4.0.6535 or later and .NET Framework 4.6.1.
+
+## How to use
+First, in your Starcounter app project, install Starcounter.Uniform trough NuGet: `Install-Package Starcounter.Uniform` and add a reference to `Starcounter.Uniform.dll`.
+
+This library provides helping method for a variety of uniform components and way to use it is slightly different for each one of them.
+
+## Uni-data-table
+`uni-data-table` is a custom element that is an adapter to `vaadin-grid`. It provides rich view of a data table which contain funcionalities like:
```

Comment:
Do people know what "adapter" means in this context? What's the difference between a wrapper and an adapter?

Reply:
I think that this explanation of `uni-data-table` should be removed from README.md in `Starcounter.Uniform`. Multiple reasons:

- is redundant in this file. That component will be documented in https://uniform.starcounter.io/UniformDocs/DataTable. Now it is documented in https://github.com/Starcounter/uniform.css/blob/master/components/uni-data-table/README.md
- nobody needs to know that `uni-data-table` is an adapter to `vaadin-grid`. Fiat does not say that [Fiat Sedici](https://en.wikipedia.org/wiki/Fiat_Sedici) is an adapter to Suzuki SX4. The whole point of rebranding is to disguise the original brand.

---

Also, in answer to @Mackiovello's questions, http://www.thecodedself.com/The-Difference-Between-an-Adapter-and-a-Wrapper/

Reply:
I agree with your comment, but I would like to save something, just to have some opening for this part of the document. I changed this beginning a little, please take a look.